cumulusci/tasks/bulkdata/generate_and_load_data.py

```diff
@@ -81,7 +89,7 @@ def _init_options(self, kwargs):
             )
 
     def _run_task(self):
-        with temporary_dir() as tempdir:
+        with TemporaryDirectory() as tempdir:
```

Comment:
temporary_dir changes to the directory and TemporaryDirectory does not. Is that a change that matters?